lib/slax/commands/github_commands.ex

```diff
@@ -48,6 +48,8 @@ defmodule Slax.Commands.GithubCommands do
       ) do
     repo = "#{org_name}/#{project_name}"
 
+    sleep(1.5)
```

Comment:
CI is failing because this function doesn't exist

Reply:
My bad no idea why I thought this repo was ruby lol.